tests/testthat/test-carehomes-support.R

```diff
@@ -495,28 +503,31 @@ test_that("Can control the seeding", {
 
   initial_y <- mod$transform_variables(initial$state)
 
-  expect_equal(initial_y$N_tot3, sum(p$N_tot))
-  expect_equal(initial_y$N_tot2, sum(p$N_tot))
+  expect_equal(initial_y$N_tot_sero_2, sum(p$N_tot))
+  expect_equal(initial_y$N_tot_PCR, sum(p$N_tot))
+  expect_equal(initial_y$N_tot_sero_2, sum(p$N_tot))
   expect_equal(initial_y$N_tot, p$N_tot)
 
   expect_equal(rowSums(initial_y$S) + drop(initial_y$I_A),
                p$N_tot)
   expect_equal(drop(initial_y$I_A),
                append(rep(0, 18), 50, after = 3))
-  expect_equal(initial_y$T_sero_pre[, 1, 1, ],
+  expect_equal(initial_y$T_sero_pre_1[, 1, 1, ],
+               append(rep(0, 18), 50, after = 3))
+  expect_equal(initial_y$T_sero_pre_2[, 1, 1, ],
                append(rep(0, 18), 50, after = 3))
   expect_equal(initial_y$T_PCR_pos[, 1, 1, ],
                append(rep(0, 18), 50, after = 3))
   expect_equal(initial_y$react_pos, 50)
 
-  ## 46 here, derived from;
+  ## 48 here, derived from;
   ## * 38 (S + N_tot)
   ## * 1 (prob_strain)
   ## * 1 (react_pos)
-  ## * 2 (N_tot2 + N_tot3)
+  ## * 3 (N_tot_sero_2 + N_tot_PCR + Ntot4)
```

Comment:
(N_tot_sero_1 + N_tot_PCR + N_tot_sero_2)?

Reply:
Corrected, and also reordered for clarity